index.html

```diff
@@ -8,34 +8,42 @@
     </head>
     <body>
         <header class="header">
-            <span class="logo-link">
+            <div class="logo-link">
                 <a href=".//index.html"><img src=".//img/logo-black.png" alt="Логотип надпись device черным цветом"></a>
-            </span>
+            </div>
             <div class="header-buttons">                
-                <div class="search">                    
+                <div class="search-container">                    
                     <label class="visually-hidden" for="input-search">Поиск по сайту</label>
                     <input class="input-search" type="text" name="search" id="input-search" placeholder="Поиск по сайту">                                    
-                    <span class="button-search"><a href="#">Найти</a></span>
+                    <div class="button-search"><a href="#">Найти</a></div>
                 </div>
                 <div class="user-buttons">
                     <ul class="user-registration">
-                        <li class="button-registration"><a href=".//registration.html">Войти</a></li>
+                        <li class="button-registration" tabindex="-1"><a href=".//registration.html">Войти</a></li>
                     </ul>
                     <ul class="user-purchase">
-                        <li class="button-compare"><a href="#">Сравнить</a></li>
+                        <li class="button-compare" tabindex="-1"><a href="#">Сравнить</a></li>
                         <li class="button-basket"><a href="#">Корзина</a></li>
                     </ul>  
                 </div>                            
             </div>
-            <nav class="nav" aria-label="Навигация по сайту">                                                
-                <ul class="nav-list">
-                    <li class="nav-catalog"><a href="#">Каталог товаров</a></li>
-                    <li class="#"><a href="#">Доставка</a></li>
-                    <li class="#"><a href="#">Гарантия</a></li>
-                    <li class="#"><a href="#">Контакты</a></li>
-                </ul>
+            <nav class="nav" aria-label="Навигация по сайту">
+                <div class="div-catalog">
+                    <ul class="ul-catalog">
+                        <li class="li-catalog"><a href="#">Каталог товаров</a></li>
```

Comment:
where are catalog items, user sees on hover?

Reply:
I don't understand how to insert it into html markdown and css